packages/selenium/test/AxeBuilderLegacyTest.cs

```diff
@@ -0,0 +1,401 @@
+using FluentAssertions;
+using Moq;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Deque.AxeCore.Commons;
+
+namespace Deque.AxeCore.Selenium.Test
+{
+    public class LegacyTestAxeScriptProvider : IAxeScriptProvider
+    {
+        public static string stubAxeScript = "stub axe script";
+        public string GetScript()
+        {
+            return stubAxeScript;
+        }
+    }
+
+    [TestFixture]
+    [NonParallelizable]
+    public class LegacyAxeBuilderTest
+    {
+        private static Mock<IWebDriver> webDriverMock = new Mock<IWebDriver>();
+        private static Mock<IJavaScriptExecutor> jsExecutorMock = webDriverMock.As<IJavaScriptExecutor>();
+        private static Mock<ITargetLocator> targetLocatorMock = new Mock<ITargetLocator>();
+        private static readonly JsonSerializerSettings JsonSerializerSettings = new JsonSerializerSettings
+        {
+            Formatting = Formatting.None,
+            NullValueHandling = NullValueHandling.Ignore
+        };
+
+        private static readonly AxeBuilderOptions stubAxeBuilderOptions = new AxeBuilderOptions
+        {
+            ScriptProvider = new LegacyTestAxeScriptProvider()
+        };
+
+        private readonly object testAxeResult = new
+        {
+            violations = new object[] { },
+            passes = new object[] { },
+            inapplicable = new object[] { },
+            incomplete = new object[] { },
+            timestamp = DateTimeOffset.Now,
+            url = "www.test.com"
+        };
+
+        [Test]
+        public void LegacyThrowWhenDriverIsNull()
+        {
+            Assert.Throws<ArgumentNullException>(() =>
+            {
+                //arrange / act /assert
+                var axeBuilder = new AxeBuilder(null, stubAxeBuilderOptions);
+                axeBuilder.Should().NotBeNull();
```

Comment:
This assertion is unnecessary/misleading, the `Assert.Throws` should be triggering on L56 alone (same comment for other `Assert.Throws` usages)